scripts/add_currency_notes.py

```
@@ -0,0 +1,194 @@
+    "SLL": "Sierra Leonean leone, Sierra Leone is a major diamond producer.",
+    "SOS": "Somali shilling, Somalia has faced decades of civil conflict.",
+    "SRD": "Surinamese dollar, Suriname is a small South American nation with gold reserves.",
+    "STN": "São Tomé and Príncipe dobra, pegged to the euro via Portugal.",
+    "SVC": "Salvadoran colón, El Salvador adopted Bitcoin as legal tender in 2021.",
+    "SYP": "Syrian pound, has collapsed dramatically due to the ongoing civil war.",
+    "SZL": "Swazi lilangeni, pegged 1:1 to the South African rand.",
+    "THB": "Thai baht, Thailand is Southeast Asia's second-largest economy.",
+    "TJS": "Tajikistani somoni, named after Ismoil Somoni, a Samanid ruler.",
+    "TMT": "Turkmenistani manat, Turkmenistan has the world's fourth-largest gas reserves.",
+    "TND": "Tunisian dinar, Tunisia is a major North African tourism destination.",
+    "TOP": "Tongan paʻanga, Tonga is a Polynesian island kingdom in the Pacific.",
```

Ruff (0.15.12)

[warning] String contains ambiguous `ʻ` (MODIFIER LETTER TURNED COMMA). Did you mean ``` (GRAVE ACCENT)?

(RUF001)